client/state/sites/constants.js

```diff
@@ -71,4 +71,5 @@ export const SITE_REQUEST_OPTIONS = [
 	'is_difm_lite_in_progress',
 	'difm_lite_site_options',
 	'site_intent',
+	'launchpad_view',
```

Comment:
@jeyip - I think in other places we're referring to this option as launchpad_screen? See D85918-code (merged) and [here](https://github.com/Automattic/wp-calypso/pull/67016) (not yet merged).

No strong feeling here on the name, but may be best for us to stay consistent. If so, could change here or in others above.